crates/rover-client/src/query/graph/push.rs

```diff
@@ -28,26 +28,26 @@ pub fn run(
     variables: push_schema_mutation::Variables,
     client: &StudioClient,
 ) -> Result<PushResponse, RoverClientError> {
+    let graph = variables.graph_id.clone();
     let data = client.post::<PushSchemaMutation>(variables)?;
-    let push_response = get_push_response_from_data(data)?;
+    let push_response = get_push_response_from_data(data, graph)?;
     build_response(push_response)
 }
 
 fn get_push_response_from_data(
     data: push_schema_mutation::ResponseData,
+    graph: String,
 ) -> Result<push_schema_mutation::PushSchemaMutationServiceUploadSchema, RoverClientError> {
     // then, from the response data, get .service?.upload_schema?
     let service_data = match data.service {
         Some(data) => data,
-        None => return Err(RoverClientError::NoService),
+        None => return Err(RoverClientError::NoService { graph }),
     };
 
     if let Some(opt_data) = service_data.upload_schema {
         Ok(opt_data)
     } else {
-        Err(RoverClientError::HandleResponse {
-            msg: "No response from mutation. Check your API key & graph name".to_string(),
-        })
+        Err(RoverClientError::MalformedResponse)
```

Comment:
```suggestion
        Err(RoverClientError::AdhocError {
            msg: "Apollo Studio failed to store the schema".to_string()
        })
```

Reply:
I just added `null_field` to `MalformedResponse` so when we tell folks to submit an issue they can tell us what field was null

Reply:
I'm really hoping these `MalformedResponse` errors just.. never occur! but since they _can_ be null, they likely will. I'd like to treat those as bugs though, so if we get an issue where folks are getting this, we need to either handle the error better or fix it in the API